atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -137,6 +138,9 @@ def run(self):
 
         if self.append:
             next_release = self.get_next_release_append(component, version, release)
+        elif is_scratch_build():
+            metadata = get_build_json().get("metadata", {})
+            next_release = metadata.get("name", "1")
```

Comment:
Haven't we talked about using a timestamp as default value?

Reply:
well we are using Openshift build name, if that isn't defined something is really off, so we just default to 1, that is really just corner case